pkg/snet/directtp/tptypes/tptypes.go

```diff
@@ -0,0 +1 @@
+package tptypes
```

Comment:
I guess we should populate this file with values from `tptypes.go.go` and remove `tptypes.go.go`